src/inspector/network_inspector.h

```diff
@@ -11,7 +11,8 @@ namespace inspector {
 
 class NetworkInspector {
  public:
-  explicit NetworkInspector(Environment* env);
+  explicit NetworkInspector(Environment* env,
+                            v8_inspector::V8Inspector* v8_inspector);
```

Comment:
Wouldn't it be safer to just use std::shared_ptr here?

Reply:
`NetworkInspector` does not own `V8Inspector`, like it does not own `Environment` as well. It can use `shared_ptr` but a raw pointer here indicates that it does not own the value.